Mathlib/MeasureTheory/Measure/Tight.lean

```diff
@@ -106,6 +107,21 @@ protected lemma inter (hS : IsTightMeasureSet S) (T : Set (Measure α)) :
     IsTightMeasureSet (S ∩ T) :=
   hS.subset inter_subset_left
 
+lemma map [OpensMeasurableSpace β] [T2Space β]
```

Comment:
Shouldn't `MeasurableSpace β` be a regular typeclass assumption here, since there is the other typeclass assumption `[OpensMeasurableSpace β]` that depends on it?